src/fruity/lockdown.vala

```diff
@@ -8,6 +8,9 @@ namespace Frida.Fruity {
 		}
 
 		private PlistServiceClient service;
+		public IOStream stream {
+			get { return service.stream; }
+		}

```

Comment:
Would be good to move this right after `device_details` (with one blank line between), as public signals/properties should be before private members.